apps/prairielearn/src/pages/error/error.js

```diff
@@ -40,7 +40,18 @@ export default function (err, req, res, _next) {
   res.send(
     ErrorPage({
       // Hide error details in production.
-      error: config.devMode ? err : { message: err.message, status: err.status },
+      error: config.devMode
+        ? err
+        : {
+            message: err.message,
+            status: err.status,
+          },
```

Comment:
Nit. It really bothers me that prettier doesn't do this by default.
```suggestion
      error: config.devMode ? err : { message: err.message, status: err.status },
```